plugins/php/php_plugin.c

```diff
@@ -332,9 +340,9 @@ PHP_FUNCTION(uwsgi_cache_clear) {
 PHP_FUNCTION(uwsgi_cache_del) {
 	
 	char *key = NULL;
-        int keylen = 0;
+        php_strlen_size keylen = 0;
```

Comment:
please fix whitespace here while you are at it :)

Reply:
Well, speaking of whitespace, I tried my best to keep them "as is", because I found that sometimes tabs are used for indentation and sometimes spaces, So what's the "canonical" indentation style for the project?

Reply:
Agree, it's a mess.  Preferred should be tabs but there's a ton of spaces around. BTW i meant to fix only that line not to clean up all the whitespace in the file.

Reply:
Okay, will fix this, but one thing to note here - if you set tabsize to 8, this is actually aligned:
![whitespace](https://user-images.githubusercontent.com/192147/72682514-f5840c80-3acd-11ea-98ed-d2d180187f07.png)

Reply:
thanks!